app/src/pages/SingleContent.vue

```diff
@@ -60,6 +62,26 @@ const is404 = computed(() => {
     return false;
 });
 
+const userPreferences = ref<{ bookmarks: { [key: string]: { ts: number } } }>({
+    bookmarks: JSON.parse(localStorage.getItem("userPreferences") || "{}").bookmarks || {},
+});
```

Comment:
Can we move this to a central location that is easily accessible in other parts of the code? The place it makes most sense according to me is in globalConfig.ts

Reply:
Yes we can do it.